### PR TITLE
feat: publish `@cartesi/rollups` through GitHub Actions

### LINKED ISSUE #51: Automate publishing package on npm

### PR DESCRIPTION
**Pull request by guidanoli**
Closes #51

### CODE REVIEW

**Review by ZzzzHui — DISMISSED** [review]

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        working-directory: onchain/rollups
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "16"
```
Comment (tuler) [inline .github/workflows/publish.yml:20]: this is rather old node. Use at least 18.

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  publish:
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        working-directory: onchain/rollups
```
Comment (tuler) [inline .github/workflows/publish.yml:11]: are you planning to remove the `onchain` directory?

File: onchain/rollups/package.json [hunk 3]
```diff
@@ -69,6 +69,7 @@
         "typescript": "^5"
     },
     "files": [
+        "abi",
```
Comment (tuler) [inline onchain/rollups/package.json:72]: which process is generating this?

Comment (guidanoli) [reply onchain/rollups/package.json:72]: `yarn prepack`

**Review by tuler — APPROVED** [review]

**Pull request merged**